diff -rupN esphttpd/mkespfsimage/heatshrink_encoder.c esphttpd.new/mkespfsimage/heatshrink_encoder.c
--- esphttpd/mkespfsimage/heatshrink_encoder.c	2015-02-03 22:17:38.000000000 +0100
+++ esphttpd.new/mkespfsimage/heatshrink_encoder.c	2015-02-03 22:24:18.000000000 +0100
@@ -1,3 +1,3 @@
 //Stupid wraparound include to make sure object file doesn't end up in heatshrink dir
 
-#include "../lib/heatshrink/heatshrink_encoder.c"
+#include "../../heatshrink/heatshrink_encoder.c"
diff -rupN esphttpd/mkespfsimage/Makefile esphttpd.new/mkespfsimage/Makefile
--- esphttpd/mkespfsimage/Makefile	2015-02-03 22:17:38.000000000 +0100
+++ esphttpd.new/mkespfsimage/Makefile	2015-02-03 22:24:08.000000000 +0100
@@ -1,5 +1,5 @@
 
-CFLAGS=-I../lib/heatshrink -std=gnu99
+CFLAGS=-I../../heatshrink -std=gnu99
 OBJS=main.o heatshrink_encoder.o
 TARGET=mkespfsimage
 
@@ -7,4 +7,4 @@ $(TARGET): $(OBJS)
 	$(CC) -o $@ $^
 
 clean:
-	rm -f $(TARGET) $(OBJS)
\ No newline at end of file
+	rm -f $(TARGET) $(OBJS)
